### DISCUSSION TITLE
Connecting trimble r4 to qfield through serial port (NMEA)

### DISCUSSION THREAD

**Discussion by dukytony** [opengisch/QField#3102] [Q&A]
Working in archaeology.

I m wondering if someone managed to get NMEA stream from an external gps rover such as trimble r4 through serial port (rs232). Would a rs232 to usb-c cable do the trick? I heard an app like gps connector can read nmea stream from usb and use it as mock location on android.

I want to test this because we work with a fixed base station on site communicating with the rover through radio. Cant use NTRIP because cell coverage is really bad on site.

Is this possible?

Thanks a lot guys!

**Comment by lucienicolier** [comment]
Hi,

You get information about using external receiver in the doc [Using an external GNSS-Receiver](https://docs.qfield.org/how-to/gnss/#using-an-external-gnss-receiver)

I hope it helps.

**Comment by dukytony** [comment]
Cant use bluetooth to get NMEA because i need trimble access connected to the R4 to get corrections from the base station
